lib/github.com/diku-dk/containers/hashkey.fut

```diff
@@ -1,26 +1,36 @@
+-- A module type for keys that can be hashed.
```

Comment:
This needs to be a doc comment (combine it with the comment below). Remember to run `futhark doc`.